app/src/main/java/org/p2p/wallet/sell/ui/payload/SellPayloadPresenter.kt

```diff
@@ -188,16 +189,18 @@ class SellPayloadPresenter(
     private fun startLoadSellQuoteJob() {
         sellQuoteJob?.cancel()
         sellQuoteJob = launch {
+            delay(SELL_QUOTE_NEW_AMOUNT_DELAY)
             while (isActive) {
                 try {
-                    delay(SELL_QUOTE_REQUEST_DEBOUNCE_TIME)
                     val sellQuote = sellInteractor.getSellQuoteForSol(
                         solAmount = selectedTokenAmount,
                         fiat = fiatCurrencyMode.toSellFiatCurrency()
                     )
                     onSellQuoteLoaded(sellQuote)
                 } catch (error: Throwable) {
                     handleError(error)
+                } finally {
+                    delay(SELL_QUOTE_REQUEST_DEBOUNCE_TIME)
```

Comment:
this will still have a delay if user will change the input, right? 
Why do we need 10 secs delay?